### PR TITLE
OSSM-7828 OSSM 2.6.3 (At Stage): [DOC] Release Notes

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
Change type: Doc update; OSSM 2.6.3 (At Stage): Release Notes

Doc JIRA: https://issues.redhat.com/browse/OSSM-7828

Fix Version: 4.14+

Doc Preview: https://84725--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes#ossm-release-2-6-3_ossm-release-notes

SME Review: @ctartici 
QE Review: @mkralik3 
Peer Review: @kcarmichael08

### CODE REVIEW

File: modules/ossm-release-2-4-12.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-4-12_{context}"]
+= {SMProductName} version 2.4.12
+
+This release of {SMProductName} is included with the {SMProductName} Operator 2.6.2, addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {product-title} 4.14 and later.
```
Comment (ctartici) [inline modules/ossm-release-2-4-12.adoc:10]: Shouldn't the operator version be 2.6.3?

**Review by ctartici — COMMENTED** [review]
Except the mentioned comments, seems all ok to me.

**Comment by gwynnemonahan** [comment]
OSSM-6267 needs to be moved from a Known issue to a Fixed issue.

**Review by ctartici — APPROVED** [review]
thanks

**Comment by shreyasiddhartha** [comment]
> [OSSM-6267](https://issues.redhat.com//browse/OSSM-6267) needs to be moved from a Known issue to a Fixed issue.

I have added it in the bug fix section of 2.5.2.

**Comment by mkralik3** [comment]
@shreyasiddhartha Shouldn't it be removed also from `ossm-rn-fixed-issues.adoc` ? ( I still see it in the known issues section, or the preview is out of date? )

**Comment by mkralik3** [comment]
LGTM

File: modules/ossm-release-2-6-3.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-6-3_{context}"]
+= {SMProductName} version 2.6.3
+
+This release of {SMProductName} updates the {SMProductName} Operator version to 2.6.3, and includes the following `ServiceMeshControlPlane` resource version updates: 2.6.3, 2.5.6 and 2.4.12.
```
Comment (kcarmichael08) [inline modules/ossm-release-2-6-3.adoc:10]:
```suggestion
This release of {SMProductName} updates the {SMProductName} Operator version to 2.6.3, and includes the following `ServiceMeshControlPlane` resource version updates: 2.6.3, 2.5.6, and 2.4.12.
```

**Review by kcarmichael08 — COMMENTED** [review]
Had a minor comment, lgtm!

**Review by maxwelldb — APPROVED** [review]

**Pull request merged**